### PR TITLE
pbTests: Allow use of default values for make-adopt-build-farm.sh

### PR DESCRIPTION
**Pull request by sxa**
Not a clue if I've stripped too much out or not, but might as well give it a shot ... Now possible thanks to https://github.com/AdoptOpenJDK/openjdk-build/pull/2167 https://github.com/AdoptOpenJDK/openjdk-build/pull/2479 https://github.com/AdoptOpenJDK/openjdk-build/pull/2482 but requires https://github.com/AdoptOpenJDK/openjdk-build/pull/2478 but weill require https://github.com/AdoptOpenJDK/openjdk-build/pull/2478 to be merged before testing

Signed-off-by: Stewart X Addison <sxa@redhat.com>

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [ ] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access) [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/1069/) [QPC](https://ci.adoptopenjdk.net/job/QEMUPlaybookCheck/208/)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Comment by Willsparker** [comment]
Looks like the build PRs blocking it are merged now, so I'll test this :-)

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@
     - role: ccache
       when: ansible_distribution != "Solaris" # Compile fails on Solaris
     - role: nasm                  # OpenJ9
-      when: ansible_architecture == 'x86_64'
+      when: ansible_architecture == 'x86_z64'
```
Comment (Willsparker) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml:69]: I'm assuming this was done due to https://adoptopenjdk.slack.com/archives/C53GHCXL4/p1614088449011500 ?
Either way, I think the cert has been renewed now (judging from the padlock in my browser).

**Review by Willsparker — CHANGES_REQUESTED** [review]
Otherwise, LGTM - I'm assuming this has already been put through VPC?

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml (hunk 1 above)
Comment (sxa) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml]: Yes. It was in a commit that had `DONOTMERGEME` in the title

Comment (Willsparker) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml]: Ah okay- I think that was overwritten by your force pushes

**Review by Willsparker — APPROVED** [review]
LGTM

**Comment by sxa** [comment]
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/1078/OS=Ubuntu1804,label=vagrant/console has passed so taking this out of draft.

**Pull request merged**